src/commands/scan.yml

```diff
@@ -76,8 +76,8 @@ steps:
           fi
           LATEST_SNYK_CLI_VERSION=$(curl https://static.snyk.io/cli/latest/version)
           echo "Downloading Snyk CLI version ${LATEST_SNYK_CLI_VERSION}"
-          curl -sO https://static.snyk.io/cli/v${LATEST_SNYK_CLI_VERSION}/snyk-<<parameters.os>>
-          curl -sO https://static.snyk.io/cli/v${LATEST_SNYK_CLI_VERSION}/snyk-<<parameters.os>>.sha256
+          curl -sO --retry https://static.snyk.io/cli/v${LATEST_SNYK_CLI_VERSION}/snyk-<<parameters.os>>
+          curl -sO --retry https://static.snyk.io/cli/v${LATEST_SNYK_CLI_VERSION}/snyk-<<parameters.os>>.sha256
```

Comment:
How many times will this retry? Isn't it worth specifying an amount, say, 3?

https://linux.die.net/man/1/curl